spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -45,5 +45,29 @@
         expect(parsed_response['errors']).to eq(['You need to sign in or sign up before continuing.']) 
       end
     end
+
+    context 'when user add more than 3 targets' do
+      before do
+        target_params[:target][:title] = "Meet rock lovers"
+        target_params[:target][:topic_id] = topic_list[1].id
+        post api_v1_targets_path, params: target_params, headers: auth_headers, as: :json
+      
+        target_params[:target][:title] = "Meet dog lovers"
+        target_params[:target][:topic_id] = topic_list[2].id
+        post api_v1_targets_path, params: target_params, headers: auth_headers, as: :json    
+
+        target_params[:target][:title] = "Meet pizza lovers"
+        target_params[:target][:topic_id] = topic_list[3].id
+        post api_v1_targets_path, params: target_params, headers: auth_headers, as: :json
```

Comment:
This is the same as 
```
let(:topic_list) { create_list(3, :topics) } 
```
Right?

If you need topics of that list belong to the same user, you can do something like 
`let(:topic_list) { create_list(3, :topics, user: some_user) } `